src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/cart.html.twig

```diff
@@ -218,11 +220,13 @@
     <td class="js-product-ref"></td>
     <td><input class="form-control js-product-unit-input" type="text"></td>
     <td>
-      <input type="number" min="1" class="form-control js-product-qty-input" style="max-width: 100px;">
-      <small class="form-text">
+      <input type="number" min="1" class="form-control js-product-qty-input" style="max-width: 100px;{% if not stockManagementEnabled %}margin-top:0px;{% endif %}">
+      {% if stockManagementEnabled %}
+      <small class="form-text">  
```

Comment:
Same here, indentation need to be fixed

Reply:
Done in the commit https://github.com/PrestaShop/PrestaShop/pull/23811/commits/18c98cabfd2ebccc51a1e2e764a9e85598eec1ef